packages/export/src/excel.js

```diff
@@ -0,0 +1,75 @@
+import _ from 'lodash/fp.js'
+import writeXlsxFile from 'write-excel-file/node'
+import { writeStreamData } from './utils.js'
+
+const convertToExcelCell = (value) => {
+  return {
+    type: String,
```

Comment:
Is there a reason we are always setting the cell type to string? According to the documentation, the following is true:

_If a cell doesn't have a type, then it is automatically detected from the value, or defaults to a String._

Reply:
there are some odd cases like zip code for instance, where some of it was treated as a number and some as a string, this changes the alignment of the values and possibly other underlying things. When I get done with the other changes I can test it again, but this seems safer atm.

Reply:
I double-checked this, and it handles it fine as it relies on the application to use all the data, I must have originally tested it with some other conditions that are no longer a concern. Updated the code with this change.

Reply:
That's fine. We can circle back to this.